draft-lenders-dns-over-coap.md

```diff
@@ -119,17 +119,13 @@ The terms "CoAP payload" and "CoAP body" are used as defined in [RFC7959].
 Selection of a DoC Server
 =========================
 
-TBD:
-
-- URI or should we rather go straight to CRI?
-- Support for more than one URI by DoC server.
-- DoC server identity, key exchange, ...
-
-URI Alternatives
-----------------
-TBD:
+It is assumed in this document that the DoC server, as well as the DNS query resource at the DoC
+server, is known to the DoC client. Possible ways to configure could be manual configuration with a
+URI {{?RFC3986}} or CRI {{?I-D.ietf-core-href}} or via a CoRE resource directory
+{{?I-D.ietf-core-resource-directory}}. Automatic configuration SHOULD only be done from a trusted
+source.
 
-- CRI {{?I-D.ietf-core-href}} or CoRAL {{?I-D.ietf-core-coral}}
+When discovering the URI through a link mechanism such as where the {{?RFC6690}} attribute "resource type" is available, the resource type "core.dns" can be used to identify a generic DNS resolver that is available to the client.
```

Comment:
When discovering the URI through a link mechanism that allows describing a resource (e.g., Resource Type attribute in  {{?RFC6690}}), the resource type "core.dns" can be used to identify a generic DNS resolver that is available to the client.

Reply:
I used "DNS resource" instead of "URI" to not exclude link mechanisms using, e.g., CRIs ;-).